src/Storage/Storage.Management/help/Az.Storage.md

```diff
@@ -134,6 +143,15 @@ Invokes failover of a Storage account.
 ### [Lock-AzRmStorageContainerImmutabilityPolicy](Lock-AzRmStorageContainerImmutabilityPolicy.md)
 Locks ImmutabilityPolicy of a Storage blob containers
 
+### [Move-AzDataLakeGen2Item](Move-AzDataLakeGen2Item.md)
+Move a file or folder to another a file or folder in same Storage account.
+
+### [New-AzDataLakeGen2Item](New-AzDataLakeGen2Item.md)
+Create a file or folder in a container.
```

Comment:
Do we align the term across all PSH cmdlets with ADLS Gen2 terms (I read from ADLS Gen2 REST API spec)? For example, should 'container' be used here or 'Filesystem', 'folder' or 'directory' should be used?

Reply:
As we discussed in the meeting with feature team and Peter, we will use the pattern Container + Path (+ StorageContext) for the cmdlets.

Reply:
It's more about the right terminology to be read by customers. At least, 'filesystem' and 'directory' is what I read from REST spec, I would assume we should keep it aligned in documentation.

Reply:
Discussed with feature team, change "Container" to "FileSystem", "Folder" to "Directory".
Design and Code are both updated. Design passed review.